adapters/smarthub/smarthub.go

```diff
@@ -60,10 +60,11 @@ func (a *adapter) getImpressionExt(imp *openrtb2.Imp) (*openrtb_ext.ExtSmartHub,
 
 func (a *adapter) buildEndpointURL(params *openrtb_ext.ExtSmartHub) (string, error) {
 	endpointParams := macros.EndpointTemplateParams{
-		Host:      params.PartnerName,
 		AccountID: params.Seat,
 		SourceId:  params.Token,
+		Host:      params.PartnerName,
```

Comment:
Might be better to delete the Host template param entirely. This is only used for endpoint macros and neither smarthub nor its aliases use this macro.

Reply:
Currently, we use both methods of redirecting traffic, both through partnerName and using aliases.We also need to use param partnerName to temporarily connect partners.

Reply:
Ah, I see. The aliases do not use this macro, but the base `smarthub` adapter does. Ok.